### DISCUSSION TITLE
Questions regarding this crate

### DISCUSSION THREAD

**Discussion by Joe23232** [fltk-rs/fltk-rs#521] [General]
Hi I have some questions regarding this crate.

1. Does this crate support Android, if I were to use something like [`android-ndk-rs`](https://github.com/rust-windowing/android-ndk-rs) crate along with this crate?
2. Is it stable or beta?
3. Is it immediate or retained mode?

**Comment by MoAlyousef** [comment]
I had a similar discussion on discord a few moments ago!
1- Yes but it's experimental. I couldn't get it to work with android-ndk-rs, I think because of conflicting implementation of Android's NativeActivity. It works with Android Studio (only tested on Windows though, but should work on other systems).
2- I wouldn't say it's stable, at least not on Android.
3- It's retained mode.

**Comment by Joe23232** [comment]
I see thanks mate, I asked this question before asking on Discord :)

**Comment by MoAlyousef** [comment]
No problem. Glad to answer here as well :)
